streams/src/test/java/org/apache/kafka/streams/integration/EosBetaUpgradeIntegrationTest.java

```diff
@@ -526,12 +521,10 @@ public void shouldUpgradeFromEosAlphaToEosBeta() throws Exception {
 
                 assignmentListener.waitForNextStableAssignment(MAX_WAIT_TIME_MS);
 
-                waitForStateTransition(stateTransitions2, CRASH);
-
                 commitErrorInjectedClient2.set(false);
                 stateTransitions2.clear();
                 streams2Alpha.close();
-                waitForStateTransition(stateTransitions2, CLOSE_CRASHED);
```

Comment:
No longer do you need to close crashed

Reply:
Why this change? client2 should transit from `PENDING_ERROR` to `ERROR` (not from `PENDING_SHUTDOWN` to `CLOSED`, and thus the new `CLOSE` check seem not to be right? Or do I miss something?

Reply:
From my understanding/ according to the new state transition diagram, if there is an error we go to `PENDING_ERROR` and if we call `close()` we transit `PENDING_ERROR -> ERROR`?

Reply:
Thanks for clarifying.